libs/clients/vmst/src/lib/utils.ts

```diff
@@ -8,39 +10,52 @@ export const createWrappedFetchWithLogging = (
   return new Promise((resolve, reject) => {
     fetch(input, init)
       .then(async (response) => {
-        const body = init?.body ? JSON.parse(init?.body as string) : {}
+        let requestBody = init?.body ? JSON.parse(init?.body as string) : {}
 
-        const newInit = {
-          ...init,
-          body: {
-            ...body,
-            applicant: 'hidden',
-            otherParentId: '' ?? 'hidden',
-            email: 'hidden',
-            phoneNumber: 'hidden',
-            paymentInfo: {
-              ...body?.paymentInfo,
-              bankAccount: 'hidden',
-            },
-            attachments: body?.attachments?.length,
+        // Filter known sensitive data
+        // TODO: Should pick what we need instead
+        requestBody = omit(requestBody, [
+          'applicant',
+          'otherParentId',
+          'email',
+          'phoneNumber',
+          'paymentInfo.bankAccount',
+          'employers.email',
+          'employers.approverNationalRegistryId',
+        ])
+
+        const vmstMetadata = {
+          request: {
+            body: requestBody,
+          },
+          response: {
+            status_text: response.statusText,
+            body: undefined,
           },
         }
+
         if (response.ok) {
-          logger.info(
-            `vmst-module.success: input - ${JSON.stringify(
-              input,
-            )}, init - ${JSON.stringify(init?.body ? newInit : init)}`,
-          )
+          logger.info(`Successfully fetched from VMST`, {
```

Comment:
Is it not possible to change the wording here? We are not always fetching from VMST